src/file.cpp

```diff
@@ -25,16 +25,27 @@
 // Default namespace for Rive Cpp code
 using namespace rive;
 
+#if !defined(RIVE_FMT_U64)
+    #if defined(__ANDROID__)
+        #define RIVE_FMT_U64 "%llu"
+        #define RIVE_FMT_I64 "%lld"
+    #else
+        #include <inttypes.h>
+        #define RIVE_FMT_U64 "%" PRIu64
+        #define RIVE_FMT_I64 "%" PRId64
+    #endif
+#endif
```

Comment:
It's what I use in my own repos.
Preferably, it should be available via some internal header, or perhaps a logging header.